In [1]:
import pandas as pd 
import numpy as np 
import pickle

In [47]:
### extra stuff

# file_path = "/data/ana/Diffuse/GlobalFit_Flavor/taupede/data/Pass2/hdf_files/NoDeepCore/HESE7/Bfr/HESE7_DoubleCascades.hdf5"
# file_path = "/data/ana/Diffuse/GlobalFit_Flavor/taupede/data/Pass2/hdf_files/WithDeepCore/HESE7/Bfr/HESE7_DoubleCascades.hdf5" # Table 7.1 Neha 
# file_path = "/data/ana/Diffuse/GlobalFit_Flavor/taupede/data/Pass2/hdf_files/HESE7_Juliana/HESE7_Juliana.hdf5"


In [48]:
file_path = "/data/ana/Diffuse/GlobalFit_Flavor/taupede/data/Pass2/hdf_files/WithDeepCore/HESE7/Spice/HESE7_DoubleCascades.hdf5" # Table 7.2


In [41]:
file = pd.HDFStore(file_path,'r')

In [42]:
import tables
# Open the HDF5 file in read-only mode
with tables.open_file(file_path, mode='r') as h5file:
    for node in h5file.walk_nodes("/", classname=None):  # None = show all node types
        print(node._v_pathname, type(node))

/ <class 'tables.group.RootGroup'>
/DeepCore_Charge <class 'tables.table.Table'>
/FinalEventClass <class 'tables.table.Table'>
/FinalTopology <class 'tables.table.Table'>
/HESEEventclass <class 'tables.table.Table'>
/HESEMillipedeFitDepositedEnergy <class 'tables.table.Table'>
/HESEMillipedeFitFitParams <class 'tables.table.Table'>
/HESEMillipedeFitTruncatedDepositedEnergy <class 'tables.table.Table'>
/HESETaupedeFitManualFitStatus <class 'tables.table.Table'>
/Homogenized_QTot <class 'tables.table.Table'>
/I3EventHeader <class 'tables.table.Table'>
/Millipede_pos <class 'tables.table.Table'>
/Monopod_pos <class 'tables.table.Table'>
/MuonTag <class 'tables.table.Table'>
/QTot <class 'tables.table.Table'>
/RecoAzimuth <class 'tables.table.Table'>
/RecoDirection <class 'tables.table.Table'>
/RecoEConfinement <class 'tables.table.Table'>
/RecoERatio <class 'tables.table.Table'>
/RecoETot <class 'tables.table.Table'>
/RecoEnergy <class 'tables.table.Table'>
/RecoL <class 'tables.table.Tab

In [49]:
file["RecoL"]

Run     Event  SubEvent  SubEventStream  exists      value
0  128619  66564632         0               0       1  26.662956
1  126283  47286594         0               0       1  16.160042
2  124625  77850022         0               0       1  10.234150
3  116698  10198436         0               0       1  14.261762
4  118607  40435683         0               0       1  12.153063
5  120844   6079235         1               0       1  11.711063
6  121240  72944671         0               0       1  17.183949

interesting event MJD 55477, run 116698, Event 10198436, length 14.261762
found in /data/ana/Diffuse/GlobalFit_Flavor/taupede/data/Pass2/TauEvents/WithDeepCore/Spice/Run00116698_MJD55477_Taupede_out.i3.bz2
using the data shovel: 
HESETaupedeFit: 14.2618
HESETaupedeFitLogl: 13677.3
HESETaupedeFitLoglNdof: 2015
RecoERatio: -0.0952822
RecoETot: 148174
RecoL: 14.2618
RecoLogE1: 4.84419
RecoLogE2: 4.92721
RecoEConfinement: 0.995958

HESEMonopodFitLogl: 13749.6
HESEMonopodFitLoglNdof: 2016

FinalTopology: 2
HESEEventclass: 2

Ook in /data/ana/Diffuse/GlobalFit_Flavor/taupede/data/Pass2/i3files/WithDeepCore/HESE7/Spice/IC79_2010/Run00116698_MJD55477_Taupede_out.i3.bz2



interesting event MJD 55477, run 116698, Event 10198436, length 23.2063
found in /data/ana/Diffuse/GlobalFit_Flavor/taupede/data/Pass2/i3files/WithDeepCore/HESE7/Bfr/IC79_2010/Run00116698_MJD55477_Taupede_out.i3.bz2
using the data shovel: 
HESETaupedeFitLogl: 13464.3
HESETaupedeFitLoglNdof: 2015
RecoERatio: 0.307079
RecoETot: 144696
RecoL: 23.2063
RecoLogE1: 4.99705
RecoLogE2: 4.72143
RecoEConfinement: 1

HESEMonopodFitLogl: 13498
HESEMonopodFitLoglNdof: 2016

FinalTopology: 1 # cascade
HESEEventclass: 2 # double
